ophyd/busy.py

```diff
@@ -0,0 +1,72 @@
+import time
+import threading
+
+from ophyd.device import Device
+from ophyd.status import DeviceStatus
+
+
+class BusyStatus(DeviceStatus):
+    """
+    A "busy" device that takes a fixed amount of time in seconds to report complete.
+
+    The clock starts as soon as the device is created (so do not hold onto these things!)
+
+    Parameters
+    ----------
+    device : Device
+        This object this status object belongs to
```

Comment:
Double "this" may be hard to read.
```suggestion
        The object this status object belongs to
```